microsoft/fast
# update csp for images hosted on placehold.it

# Pull Request

## 📖 Description
Fixes Cors Security Policy (CSP) for image sources hosted on placehold.it.  



### 🎫 Issues
Closes #4820 


## 👩‍💻 Reviewer Notes
This PR has to go in before we can validate and test. I have manually validated the configuration directly on https://stage.create.fast.design and it matches the changes being submitted here. 



## 📑 Test Plan
1. Load https://create.fast.design
2. Open developer tools
3. Add a Card to the interface
4. View network trace for errors
5. Images pulled from placehold.it should load successfully with 200 status code.


## ✅ Checklist

### General
